### PR TITLE
Run ESLint against all the JS files

### PR DESCRIPTION
**Pull request by pdehaan**
This will also make sure the /stories/* are linted as well as /*.js files.

¯\\\_(ツ)\_/¯

### CODE REVIEW

File: stories/index.js [hunk 1]
```diff
@@ -3,8 +3,8 @@ import { storiesOf, action, linkTo } from '@kadira/storybook';
 import { host } from 'storybook-host';
 
 import SnoozePopup from '../src/lib/components/SnoozePopup';
-import MainPanel from '../src/lib/components/MainPanel';
-import ManagePanel from '../src/lib/components/ManagePanel';
+// import MainPanel from '../src/lib/components/MainPanel';
+// import ManagePanel from '../src/lib/components/ManagePanel';
```
Comment (bwinton) [inline stories/index.js]: Why are these commented out (instead of just removed)?

Comment (pdehaan) [inline stories/index.js]: I flipped a coin. I wasn't sure if you were planning on adding more stories or tests for this, and that's why they were imported initially.

But it's easy enough for me to amend the commit and delete the two lines. Your call.

Comment (bwinton) [inline stories/index.js]: I'll merge whatever you have up at the end of this meeting.  Either would be fine.  🙂

Comment (pdehaan) [inline stories/index.js]: Deleted and force pushed. We can always dig through Git history if we need to recreate those two import statements. xoxo

**Pull request merged**